mautic/mautic
# Placeholder shown for Timezone field is improper

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | Y
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
The issue is that the 'Preferred Timezone' field label is not translated.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Login to Mautic.
2. Navigate to Contacts and click '+New'.
3. Scroll down the form and observe the 'Preferred Timezone' field's label.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Navigate to Contacts and click '+New'.
3. Scroll down the form and observe the 'Preferred Timezone' field's label.
